ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/ccj/CountyCourtJudgmentMapper.java

```diff
@@ -0,0 +1,71 @@
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.ccj.PaymentSchedule;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+
+import java.util.Optional;
+
+import static uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption.valueOf;
+
+@Component
+public class CountyCourtJudgmentMapper implements Mapper<CCDCountyCourtJudgment, CountyCourtJudgment> {
+
+    @Override
+    public CCDCountyCourtJudgment to(CountyCourtJudgment countyCourtJudgment) {
+
+        CCDCountyCourtJudgment.CCDCountyCourtJudgmentBuilder builder = CCDCountyCourtJudgment.builder();
+
+        countyCourtJudgment.getDefendantDateOfBirth().ifPresent(builder::defendantDateOfBirth);
+        countyCourtJudgment.getPaidAmount().ifPresent(builder::paidAmount);
+        builder.paymentOption(valueOf(countyCourtJudgment.getPaymentOption().name()));
+        countyCourtJudgment.getRepaymentPlan().ifPresent(repaymentPlan -> {
+            builder.repaymentPlanFirstPaymentDate(repaymentPlan.getFirstPaymentDate());
+            builder.repaymentPlanInstalmentAmount(repaymentPlan.getInstalmentAmount());
+            builder.repaymentPlanPaymentLength(repaymentPlan.getPaymentLength());
+            builder.repaymentPlanPaymentSchedule(CCDPaymentSchedule.valueOf(repaymentPlan.getPaymentSchedule().name()));
+            builder.repaymentPlanCompletionDate(repaymentPlan.getCompletionDate());
+        });
+        countyCourtJudgment.getPayBySetDate().ifPresent(builder::payBySetDate);
+        countyCourtJudgment.getStatementOfTruth()
+            .ifPresent(sot -> {
+                builder.statementOfTruthSignerName(sot.getSignerName());
+                builder.statementOfTruthSignerRole(sot.getSignerRole());
+            });
+        Optional.ofNullable(countyCourtJudgment.getCcjType()).ifPresent(ccjType ->
+            builder.ccjType(CCDCountyCourtJudgmentType.valueOf(ccjType.name()))
+        );
+
+        return builder.build();
+    }
+
+    @Override
+    public CountyCourtJudgment from(CCDCountyCourtJudgment ccdCountyCourtJudgment) {
+
+        return new CountyCourtJudgment(
+            ccdCountyCourtJudgment.getDefendantDateOfBirth(),
+            PaymentOption.valueOf(ccdCountyCourtJudgment.getPaymentOption().name()),
+            ccdCountyCourtJudgment.getPaidAmount(),
+            RepaymentPlan.builder().paymentLength(ccdCountyCourtJudgment.getRepaymentPlanPaymentLength())
+                .instalmentAmount(ccdCountyCourtJudgment.getRepaymentPlanInstalmentAmount())
+                .firstPaymentDate(ccdCountyCourtJudgment.getRepaymentPlanFirstPaymentDate())
+                .completionDate(ccdCountyCourtJudgment.getRepaymentPlanCompletionDate())
+                .paymentSchedule(
+                    PaymentSchedule.valueOf(ccdCountyCourtJudgment.getRepaymentPlanPaymentSchedule().name()))
+                .build(),
+            ccdCountyCourtJudgment.getPayBySetDate(),
```

Comment:
but still it will be null while creating the object. Repayment plan will be now built only if the value from CCJ object is available. Added extra null check